spaceship/lib/spaceship/update_checker.rb

```diff
@@ -0,0 +1,39 @@
+module Spaceship
+  class UpdateChecker
+    UPDATE_URL = "https://fastlane-refresher.herokuapp.com/spaceship"
+
+    def self.ensure_spaceship_version
+      return if defined?(SpecHelper) # is this running via tests
+      return if ENV["FASTLANE_SKIP_UPDATE_CHECK"]
+
+      require 'faraday'
+      require 'json'
+
+      response = Faraday.get(UPDATE_URL)
+      return if response.nil? || response.body.to_s.length == 0
+
+      version = JSON.parse(response.body)["version"]
```

Comment:
Just curious here: why don't we check RubyGems for the most up to date version here instead of _refresher_?